pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -447,7 +447,7 @@ stdenv.mkDerivation (finalAttrs: {
      */
     (enableFeature withAlsa "alsa")
     # FIXME: see if jellyfin-ffmpeg is already on a version >= 6.1 to use enableFeature
-    (optionalString withAribcaption "--enable-libaribcaption")
+    (optionalString (withAribcaption && lib.versionAtLeast finalAttrs.version "6.1") "--enable-libaribcaption")
```

Comment:
Consider duplicating this change on the later `buildInputs` reference.

https://github.com/NixOS/nixpkgs/blob/66afe6cefc3909b0a6a4b78ab47e8908507d4886/pkgs/development/libraries/ffmpeg/generic.nix#L566

Reply:
Good point. I'll do that should my refactor preventing this issue in the first place fail. Let's merge this first to unbreak jellyfin.